pax-web-tomcat/src/main/java/org/ops4j/pax/web/service/tomcat/internal/TomcatServerWrapper.java

```diff
@@ -271,23 +271,7 @@ private OsgiExistingStandardWrapper(Servlet existing, ServletModel model) {
 		@Override
 		public synchronized void load() throws ServletException {
 			try {
-				instance = ContextClassLoaderUtils.doWithClassLoader(
-						model.getContextModel().getClassLoader(),
-						new Callable<Servlet>() {
-
-							@Override
-							public Servlet call() {
-								try {
-									return loadServlet();
-								} catch (final ServletException e) {
-									LOG.warn(
-											"Caucht exception while loading Servlet with classloader {}",
-											e);
-									return null;
-								}
-							}
-
-						});
+				instance =  loadServlet();
```

Comment:
we need to watch if there are some regressions with other tests, but the ResourceDelegatingBundleClassLoader might already do the trick here.